src/App.js

```diff
@@ -1,5 +1,5 @@
-import { Route, Routes } from "react-router-dom";
-import { useContext,useEffect } from "react";
+import { Route, Routes, useNavigate } from "react-router-dom";
+import { useContext, useEffect } from "react";
 import "./App.css";
 import Header from "./components/Header/Header";
```

Comment:
use Barrel imports so that you dont have to import components seperately. Instead export it to index.js file and then use these from here

Reply:
Yes , I am planning to change the folder structure and imports part soon.